### ISSUE TITLE
Custom title bar

### ISSUE THREAD

**Issue by Frusasoft-frusadev** [vlang/ui#491]
Would be great if you could add the ability to customize the title bar with html, or just be able to customize it.

**Comment by medvednikov** [comment]
You mean changing window's title after creation?

**Comment by athrvvvv** [comment]
> You mean changing window's title after creation?

It would be freezing cool 🥶😎

**Comment by Frusasoft-frusadev** [comment]
No, i mean customizing the title bar. The same way electron does
